lib/glific/jobs/minute_worker.ex

```diff
@@ -134,7 +134,7 @@ defmodule Glific.Jobs.MinuteWorker do
         Partners.perform_all(&FlowContext.wakeup_flows/1, nil, [])
 
       "execute_triggers" ->
-        Triggers.execute_triggers()
+        Triggers.periodic_trigger(Partners.active_organizations([]) |> Map.keys())
```

Comment:
```suggestion
        Partners.perform_all(&Triggers.execute_triggers/2, nil, [])
```

Reply:
Also, let's add logs while executing the triggers.  That will help us to test and everything.